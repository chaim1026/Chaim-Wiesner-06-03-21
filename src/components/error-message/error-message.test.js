import React from 'react';
import { shallow } from 'enzyme';
import ErrorMessage from './error-message.component';

it('should render ErrorMessage component', () => {
    expect(shallow(<ErrorMessage />)).toMatchSnapshot();
});