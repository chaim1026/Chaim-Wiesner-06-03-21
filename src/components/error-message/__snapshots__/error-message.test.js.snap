// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ErrorMessage component 1`] = `ShallowWrapper {}`;
